src/hooks/semesters/use-semesters.hook.ts

```diff
@@ -7,7 +7,7 @@ import { useRecoilCallback } from 'recoil'
 
 type HandleRefresh = () => Promise<GetSemestersResDTO>
 export const useSemesters = () => {
-  const onSemesters: HandleRefresh = useRecoilCallback(
+  const onGetSemesters: HandleRefresh = useRecoilCallback(
```

Comment:
HandleRefresh => OnGetSemesters

Reply:
Handled in [6015b91](https://github.com/ajktown/wordnote/pull/158/commits/6015b9172668f39d6487836b62c2d30bdb1bf421)